### ISSUE TITLE
typo on F24?

### ISSUE THREAD

**Issue by momdo** [w3c/wcag#281]
in <a href="https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F24#F24-failex2">Example Code of Failure Example 2</a>:

> &lt;title>Setting the canvas background&lt;/title>

I feel "Setting the canvas *foreground*" is correct.

**Comment by alastc** [comment]
Corrected, thank you. 
It should appear in the next publishing run, probably Friday (late US eastern time).
